packages/react/src/cerbos-provider.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Client, ClientWithPrincipal } from "@cerbos/core";
+// eslint-disable-next-line import/no-extraneous-dependencies -- TODO: not sure about this rule, maybe importing from peer deps should be allowed?
+import type { ReactElement, ReactNode } from "react";
```

Comment:
not sure about this rule, maybe importing from peer deps should be allowed?

Reply:
Yeah, agreed, it should be allowed. None of the other packages have peer dependencies so I never ran into this before.